test/integration/submission/submission.spec.ts

```diff
@@ -802,6 +799,108 @@ describe('Submission Api', () => {
     });
   });
 
+  describe('clinical-submission: commit', function() {
```

Comment:
should we group the http code checks in one place and keep clinical-submission stuff in another?